docs/api/field_type_reference.md

```diff
@@ -1435,6 +1436,61 @@ ezpublish:
     The option to automatically get user coordinates through the "Locate me" button
     is only available when the back office is served through the `https://` protocol.
 
+## Matrix Field Type
+
+This Field represents and handles a table of rows and columns of data.
+
+| Name     | Internal name | Expected input |
+|----------|---------------|----------------|
+| `Matrix` | `ezmatrix`    | `array`        |
+
+The Matrix Field Type is available via the eZ Platform Matrix Bundle
+provided by the https://github.com/ezsystems/ezplatform-matrix-fieldtype package.
+
+### PHP API Field Type
+
+#### Input expectations
+
+|Type|Description|Example|
+|------|------|------|
+|`array`|array of `EzSystems\EzPlatformMatrixFieldtype\FieldType\Value\Row` objects which contain column data|see below|
+
+Example of input:
+
+```php
+new FieldType\Value([
+    new FieldType\Value\Row(['col1' => 'Row 1, Col 1', 'col2' => 'Row 1, Col 2']),
+    new FieldType\Value\Row(['col1' => 'Row 2, Col 1', 'col2' => 'Row 2, Col 2']),
+    new FieldType\Value\Row(['col1' => 'Row 3, Col 1', 'col2' => 'Row 3, Col 2']),
+]);
+```
+
+#### Value Object
+
+`EzSystems\EzPlatformMatrixFieldtype\FieldType\Value` offers the following properties:
+
+|Property|Type|Description|
+|------|------|------|
+|`rows`|`RowsCollection`|Array of `Row` objects containing an array of cells (`Row::getCells()` returns array `['col1' => 'Value 1', /* ... */]`)|
+
+#### Validation
+
+The minimum number of rows is set on Content Type level for each Field.
+
+Validation checks for empty rows. A row is considered empty if it contains only empty cells (or cells containing only spaces). Empty rows are removed.
+
+If after removing empty rows the number of rows does not fulfill the configured `Minimum number of rows`, the Field will not validate.
```

Comment:
**optional:** insert commas:
If, after removing empty rows, the numbers of rows...